miranda/validators.py

```diff
@@ -107,7 +101,7 @@
     {
         "type": "simulation",
         "processing_level": Or(*PROCESSING_LEVELS),
-        Optional("project"): str,
+        "activity": str,
         Or("institution", "driving_institution"): Or(*INSTITUTIONS),
         "source": str,
         "domain": str,
```

Comment:
`bias_adjust_institution` should be removed

Reply:
It doesn't need to be removed _from the validators_. Just because the file structure we've landed on doesn't use this field, I'd still like to see it be parsed and available among the facets, as that can aid in searching for data in the future (i.e. not using `xscen`).

Reply:
ok!